core/App/ehsm_napi.cpp

```diff
@@ -1380,13 +1389,16 @@ char *NAPI_RA_GET_API_KEY(const char *p_att_result_msg)
     retJsonObj.addData_uint8Array("nonce", pt_att_result_msg->platform_info_blob.nonce.rand, 16);
     retJsonObj.addData_string("appid", p_appid);
     retJsonObj.addData_string("apikey", (char*)p_apikey.data);
```

Comment:
Do we still need to return the plaintext of apikey?

Reply:
nodejs  store it to db

Reply:
Clear. Thanks.